scylla-cdc-replicator/src/main/java/com/scylladb/cdc/replicator/operations/preimage/PreImageOperationHandler.java

```diff
@@ -121,7 +126,7 @@ private static Object asObject(Cell preimageCell) {
     // hack: we're using ChangeSchema.DataType to represent base table types
     private static ChangeSchema.DataType calculateBaseType(ChangeSchema.ColumnDefinition colDef) {
```

Comment:
`calculateBaseType` could be reduced to just `colDef.getBaseTableDataType()`, but... currently `baseTableDataType` can be not consistent with `cdcLogDataType`... (next PRs will fix this and then this method could be removed)